GNUmakefile

```diff
@@ -31,7 +31,7 @@ MAKES-REALCLEAN := $(dir $(YTS-DIR))
 
 # Setup and include go.mk and shell.mk:
 GO-FILES := $(shell find -not \( -path ./.cache -prune \) -name '*.go' | sort)
-GO-CMDS-SKIP := test fmt vet
```

Comment:
Please check the behavior of this change.

I feel like it would enable go vet because the Makefile would make it available

https://github.com/makeplus/makes/blob/main/go.mk

I wouldn't change this.

This variable is here to allow to overwrite make targets that are available by default via makeplus.

Reply:
Done